index.csv

```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+rework,MIT,1.0.1
```

Comment:
no version of rework contains a license declaration or a license file that i can see; is there a link to somewhere the author has explicitly stated that a license should retroactively apply?

Reply:
It's in the Readme, isn't it?

Reply:
If we don't allow Readme file entry for license let me know, I will remove `rework` from here

Reply:
aha, thanks - https://github.com/reworkcss/rework#license (also available in the npm tarballs).

In that case, let's also add entries for v1.0.0, v0.20.3, v0.20.2, v0.20.1, v0.20.0, v0.19.0, v0.18.3, v0.18.2, v0.18.1, v0.18.0, v0.17.4, v0.17.3, v0.17.2, v0.17.1, v0.17.0, v0.16.0, v0.15.0, v0.14.0, v0.13.2, v0.13.1, v0.13.0, v0.12.2, v0.12.1, v0.12.0, v0.11.0, v0.10.2, 0.10.1, v0.10.0, v0.9.2, v0.9.1, v0.9.0, v0.8.0, v0.7.0, v0.6.1, v0.6.0, v0.5.1, v0.5.0, v0.4.1, v0.4.0, v0.3.0, v0.1.2, v0.1.1, v0.1.0, and v0.0.1.